cluster/cluster_test.go

```diff
@@ -248,6 +248,25 @@ var _ = Describe("cluster/cluster tests", func() {
 			Expect(commandMap[3]).To(Equal([]string{"ssh", "-o", "StrictHostKeyChecking=no", "testUser@remotehost2", "echo 3"}))
 		})
 	})
+	Describe("GenerateLocalCommandMapForSegments", func() {
+		It("Returns a map of local commands for one segment, including master", func() {
+			testCluster := cluster.NewCluster([]cluster.SegConfig{masterSeg, remoteSegOne})
+			commandMap := testCluster.GenerateLocalCommandMapForSegments(true, func(id int) string {
+				return fmt.Sprintf("echo %d", id)
+			})
+			Expect(len(commandMap)).To(Equal(2))
+			Expect(commandMap[-1]).To(Equal([]string{"bash", "-c", "echo -1"}))
+			Expect(commandMap[1]).To(Equal([]string{"bash", "-c", "echo 1"}))
+		})
+		It("Returns a map of local commands for one segment, excluding master", func() {
```

Comment:
This test has the same description as the one above it.

Reply:
the top one includes master, and this one excludes master.

Returns a map of local commands for one segment, **including** master
Returns a map of local commands for one segment, **excluding** master